apps/chat/src/components/Common/ItemContextMenu.tsx

```diff
@@ -119,6 +122,13 @@ export default function ItemContextMenu({
         Icon: IconCopy,
         onClick: onDuplicate,
       },
+      {
+        name: t('View'),
+        display: !!onView && isExternal,
+        dataQa: 'view',
+        Icon: IconEye,
+        onClick: onView,
```

Comment:
React Hook useMemo has a missing dependency: 'onView'. Either include it or remove the dependency array